packages/xchain-thorchain/src/client.ts

```diff
@@ -242,36 +216,6 @@ class Client implements ThorchainClient, XChainClient {
     return getExplorerTxUrl({ urls: this.explorerUrls, network: this.network, txID })
   }
 
-  /**
-   * Set/update a new phrase
-   *
-   * @param {string} phrase A new phrase.
-   * @returns {Address} The address from the given phrase
-   *
-   * @throws {"Invalid phrase"}
-   * Thrown if the given phase is invalid.
-   */
-  setPhrase(phrase: string, walletIndex = 0): Address {
-    if (this.phrase !== phrase) {
-      if (!xchainCrypto.validatePhrase(phrase)) {
-        throw new Error('Invalid phrase')
-      }
-      this.phrase = phrase
-    }
-
-    return this.getAddress(walletIndex)
-  }
-
-  /**
-   * Get getFullDerivationPath
-   *
-   * @param {number} index the HD wallet index
-   * @returns {string} The bitcoin derivation path based on the network.
-   */
-  getFullDerivationPath(index: number): string {
```

Comment:
Why the removement of `getFullDerivationPath`?

Reply:
it's removed because this class now extends the BaseXChainClient, which implements getFullDerivationPath